06-building-a-program-start-to-finish/solving-the-problem/output/inverted.js

```diff
@@ -0,0 +1,231 @@
+/**
+ *
+ * @class InvertedIndex
+ */
+class InvertedIndex {
+
+  /**
+   * Creates an instance of InvertedIndex.
+   * @memberOf InvertedIndex
+   */
+  constructor() {
+    this.searchIndices = {};
+    this.indexedFiles = {};
+  }
+
+  /**
+   *
+   *
+   * @param {Object} eachFile - Javascript object containing file properties
+   * @returns {Object} fileContent - content of the read file
+   *
+   * @memberOf InvertedIndex
+   */
+  readFile(eachFile) {
```

Comment:
I have now added static to readFile method and removed "class-methods-use-this": 0,
from my .eslintrc.json file.